spec/components/work_package_relations_tab/relations_mediator_spec.rb

```diff
@@ -94,5 +129,33 @@
         end
       end
     end
+
+    describe "#all_relation_items" do
+      let(:work_package) { build_stubbed(:work_package) }
+
+      it "returns all relations of the group as RelationItem instances, " \
+         "ordered by oldest first (lowest id first), mixing visible and ghost relations" do
+        relation_group = described_class::RelationGroup.new(
+          type: "relates",
```

Comment:
would it be more consistent to use the constant here?

```suggestion
          type: Relation::TYPE_RELATES,
```

Reply:
yes, definitely.
Actually, I changed "children" to "child", then I looked into `Relation` to see how to implement the order and saw that constant `TYPE_CHILD`, so I thought I should use it, and then the editor suggested to replace more of them which seemed like a good idea, but I did not replace them all.

That's good that you spotted this one.